src/pybind/mgr/progress/module.py

```diff
@@ -168,6 +168,9 @@ def set_failed(self, message):
         self._progress = 1.0
         self._failed = True
         self._failure_message = message
+
```

Comment:
is removing the _refresh() call from set_failed() intended?

Reply:
@jdurgin 
nope, I think I made a mistake